core/network/types/collator_messages.hpp

```diff
@@ -182,40 +182,251 @@ namespace kagome::network {
     Signature signature;
   };
 
-  struct Seconded {
+  struct SignedStatement {
     SCALE_TIE(2);
 
     primitives::BlockHash relay_parent;  /// relay parent hash
     Statement statement;                 /// statement of seconded candidate
   };
 
+  /// Seconded is an alias for signed statement.
+  using Seconded = SignedStatement;
+
+  struct BitfieldData {
```

Comment:
Better move non message types for reuse to "parachain/types.hpp" into "namespace kagome::parachain"?
```suggestion
  struct SignedBitfield {
```